src/components/config_profile/test/smartDeviceLink.ini

```diff
@@ -168,6 +168,10 @@ FrequencyTime = 1000
 ApplicationListUpdateTimeout = 2
 ; Max allowed threads for handling mobile requests. Currently max allowed is 2
 ThreadPoolSize = 1
+; Menu title default text
+MenuTitle = MENU
```

Comment:
@dtrunov The [INI file][ApplicationManager] section was in requirements for both new fields ([APPLINK-22706](https://adc.luxoft.com/jira/browse/APPLINK-22706) and [APPLINK-22707](https://adc.luxoft.com/jira/browse/APPLINK-22707))